docs/source/usersguide/random_ray.rst

```diff
@@ -0,0 +1,303 @@
+
+To help the user set this parameter, OpenMC will report the average flat source region miss rate at the end of the simulation. Additionally, OpenMC will alert the user if very high miss rates are detected, so that they are aware that more rays and/or more active ray length might improve numerical performance. Thus, a "guess and check" approach to this parameter is recommended, where a very low guess is made, a few iterations are performed, and then the user restarts the simulation with a larger value until the "low ray density" messages go away.
+
+.. note::
+    In summary, the user should select an inactive length corresponding to many times the mean free path of a particle O(10 - 100cm) to ensure accuracy of the starting angular flux. The active length should be 10x the inactive length to amortize its cost. The number of rays should be enough so that nearly all FSRs are hit at least once each power iteration (the hit fraction is reported by OpenMC for empirical user adjustment).
+
+.. warning::
+    For simulations where long range uncollided flux estimates need to be accurately resolved (e.g., shielding, detector response, problems with significant void areas), make sure that selections for inactive and active ray lengths are sufficiently long to allow for transport to occur between source and target regions of interest. 
+
+----------
+Ray Source
+----------
+
+Random ray requires that the ray source be uniform in space and angle, throughout the entire phase space of the simulation. To facilitate sampling, the user must specify a single random ray source for sampling rays in both eigenvalue and fixed source solver modes. The random ray integration source should be of type :class:`openmc.IndependentSource`, and is specified as part of the ``settings.random_ray`` dictionary. Note that the source must be isotropic, and not limited to only fissionable regions. Additionally, the source box must cover the entire simulation domain. In the case of a simulation domain that is not box shaped, a box source should still be used to bound the domain but with the source limited to rejection sampling the actual simulation universe (which can be specified via the ``domains`` field of the :class:`openmc.IndependentSource` Python class). Similar to Monte Carlo sources, for 2D problems (e.g., a 2D pincell) it is desireable to make the source bounded near the origin of the infinite dimension. An example of an acceptable ray source for a 2D 2x2 lattice would look like:
```

Comment:
OK, this clarifies my previous question. But doesn't it only apply to eigenvalue mode?

Reply:
I've added a RST "note" here to call clarify that the random ray source is not related to any physical particle sources (and thus still needs to be supplied and to meet the uniform/isotropic conditions regardless of eigenvalue vs. fixed source mode).